server/views/partials/layout.njk

```diff
@@ -50,6 +51,12 @@
 
 {% block header %}
   {% include "./header.njk" %}
+  {{ 
```

Comment:
I think you may need to wrap this in `{% if model.prisonerDetail %}`.

There aren't any service pages that don't have a prisoner in context. But we still have a `Service under construction` page if you go to the index route of the service. (Although this may be replaced with a redirect or something else.)

Reply:
Or not sure if the `?` notation works in nunjucks. `model?.prisonerDetail?.offenderNo`

Reply:
this is an up in the air situation at the moment.

The component can handle being within and not within a prisoner context (the urls for reference https://github.com/ministryofjustice/hmpps-design-system-frontend/blob/main/src/hmpps/components/legal-information/service-header/template.njk#L4-L17) and the same can be said of the CCRD service with the index route (currently there is a index route but that will be removed for MVP and only left with an overview page)